fuzzingandroid/FuzzerEngine/fuzzerengine/coverage_manager.py

```diff
@@ -5,7 +5,9 @@
 import time
 import threading
 import re
-from config_fuzzer import RunParameters 
+from xml.dom import minidom
+from xml.parsers.expat import ExpatError
+from config_fuzzer import RunParameters
```

Comment:
All changes in this file are to add Jacoco as coverage tool. 
@Zhendong2050: PR into TM?